Apps/APN/ios/MyAppPushNotificationsHandler.swift

```diff
@@ -1,25 +1,22 @@
 import Foundation
 import CioMessagingPushAPN
-import UserNotifications
 import CioTracking
 
 @objc
 public class MyAppPushNotificationsHandler : NSObject {
 
   public override init() {}
 
-  @objc(setupCustomerIOClickHandling:)
-  public func setupCustomerIOClickHandling(withNotificationDelegate notificationDelegate: UNUserNotificationCenterDelegate) {
+  @objc(setupCustomerIOClickHandling)
+  public func setupCustomerIOClickHandling() {
     // This line of code is required in order for the Customer.io SDK to handle push notification click events.
     // We are working on removing this requirement in a future release.
     // Remember to modify the siteId and apiKey with your own values.
     CustomerIO.initialize(siteId: Env.siteId, apiKey: Env.apiKey, region: .US) { config in
       config.autoTrackDeviceAttributes = true
       config.logLevel = .debug
     }
-    
-    let center  = UNUserNotificationCenter.current()
-    center.delegate = notificationDelegate
+    MessagingPushAPN.initialize(configOptions: nil)
```

Comment:
Previously `MessagingPush.initialize` was optional too, if is a requirement now, maybe need to add a comment that states that or vice versa

Reply:
To make use of the new features such as automatic fetching of device token etc, the customer will be required to initialise the module (APN/FCM) depending on their project requirement. However, I think it would be great if we add a comment here mentioning that this configuration is available only starting from version `X.Y.Z`

Reply:
+1 to what Aman said. 

`MessagingPush.initialize` is optional, but in order for customers to use any of the new push features we have been creating (auto device token, auto push click handling), it's required to add.

Reply:
I'll add suggested comments. Good ideas.

Reply:
I do not currently agree with adding comments to this file in the sample app because this file is a copy/paste from the official docs. 

[I added a commit](https://github.com/customerio/customerio-reactnative/pull/225/commits/86395203db39b447bae7529f2533f1f07457ae72) which does add some comments that could be helpful in this file. 

Does that satisfy what you two are considering?

Reply:
I personally, believe this comment might be really helpful, a lot of times users have picked and chose the things they liked from our sample app and docs. Which is why you would see them not initializing CIO in NSE and a lot more similar examples. 

This line is required for our new feature to work and for us to have less CS tickets so I would want to ensure, readers know this too. 

But if most of the folks feel it's okay not to specify it, we can go ahead without the comment. 👍

Reply:
Thanks Levi for adding a comment on the file. I would still favour adding a comment on this particular line of code since this is a new addition to what we have been doing and this could easily be missed in case we do not highlight the need of calling this code.

` MessagingPushAPN.initialize(configOptions: nil)`